reporters/SponsorReporter.js

```diff
@@ -33,15 +31,22 @@ class SponsorReporter {
       italic("- Each monitored wallet within the configuration object has their position and token balances printed")
     );
 
+    // Define the table rows beforehand to re-use the variables.
+    const row1Name = `Synthetic debt(${this.empProps.syntheticCurrencySymbol})`;
+    const row2Name = `Backing collateral${this.empProps.collateralCurrencySymbol})`;
+    const row3Name = "Position CR %";
+    const row4Name = `Synthetic balance(${this.empProps.syntheticCurrencySymbol})`;
+    const row5Name = `Collateral balance(${this.empProps.collateralCurrencySymbol})`;
+    const row6Name = "ETH balance";
+
     // Place holder object to store all table information.
-    let tableInfo = {
-      "Token debt": {},
-      "Backing collateral": {},
-      "Position CR %": {},
-      "Synthetic balance": {},
-      "Collateral balance": {},
-      "ETH balance": {}
-    };
+    let tableInfo = {};
+    tableInfo[row1Name] = {};
```

Comment:
If you define an array of string row names, you could do `rowNames.forEach(row => tableInfo[row] = {})`

Reply:
+1, that might be a little cleaner. It's annoying that js doesn't allow you to dynamically generate keys in place.

Reply:
ye, this will be much cleaner. will give this refactor a go :)